releasenotes/notes/process-agent-service-tag-extraction-3fb97205155ed1a4.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    Add service tag extraction based on a process's command line args for service monitoring. This feature is configured
```

Comment:
```suggestion
    Add service tag extraction based on a process's command line arguments for service monitoring. This feature is configured
```

Reply:
I know I didn't add this the first time around, but on second thought, probably best to use the full word. :)